marlowe-tutorial/doc/default.nix

```diff
@@ -0,0 +1,16 @@
+{ stdenv, lib, asciidoctor, python2, playgroundUrl ? null, haddockUrl ? null }:
```

Comment:
There is no haddock, we should just get rid of this.

Reply:
this is for the plutus playground

Reply:
I mean `haddockUrl`. It's not set, and there's nothing to set it to. It's just a leftover copied from the plutus one.

Reply:
ah, sorry